pkgs/applications/audio/drumgizmo/default.nix

```diff
@@ -7,7 +7,7 @@ stdenv.mkDerivation rec {
   name = "drumgizmo-${version}";
 
   src = fetchurl {
-    url = "http://www.drumgizmo.org/releases/${name}/${name}.tar.gz";
+    url = "https://www.drumgizmo.org/releases/${name}/${name}.tar.gz";
```

Comment:
Since we have these are fixed output, we shouldn't need https, right? Maybe we can just leave these alone and just make changes to homepage (unless repology.org is looking at these urls too).